en/docs/guides/identity-federation/google.md

```diff
@@ -55,14 +55,122 @@ This guide assumes you have your own application. If you wish to try out this fl
 
 ## Configure Google as a SAML IdP in WSO2 IS
 
-(insert-admin-portal-link)
+1.  Start WSO2 Identity Server by navigating to
+    `           <IS_HOME>/bin/          ` folder and running the
+    following command.
+
+    -   **Windows**
+		``` java
+		.\wso2server.bat
+		```
+    -   **Linux/Unix**
+    	``` java
+    	sh wso2server.sh
+   	 	```
+
+2.  Log in to the management console (`https://<IS_HOST>:<PORT>/carbon`) using admin/admin credentials.
+3.  Click **Add** under **Identity Providers** on the **Main** tab.
+4.  Enter "GoogleIdP" as the identity provider name and expand
+    **Federated Authenticators\>SAML2 Web SSO Configuration**.
+5.  Select **Enable SAML2 Web SSO** and enter "wso2is1" as the **Service
+    Provider Entity ID**.
+
+    !!! info 
+		The **Service Provider Entity ID** value should be equal to the
+		value you entered as the Entity ID when configuring Google.
+
+6.  Select **Metadata File Configuration** as the **Mode** and upload
+    the `          GoogleIDPMetadata         `.xml file you downloaded
+    earlier.  
+    ![metadata-file-config](../../../assets/img/guides/metadata-file-config.png)
+    
+7.  Click **Register** to save the identity provider configurations.
+8.  Once the IdP is registered, it is listed under Identity
+    Providers. Click **Edit** and expand **Federated
+    Authenticators\>SAML2 Web SSO Configuration**.  
+    ![idp-list](../../../assets/img/guides/idp-list.png)
+    
+9.  Select **Enable Logout** and enter "
+    https://accounts.google.com/logout" as the Logout URL.
+    ![enter-the-logout-url](../../../assets/img/guides/enter-the-logout-url.png)
+    
+10. Click **Update** to save the changes.
 
 ---
 
 ## Configure the service provider
 
-(insert-admin-portal-link)
+1.  Access the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`).
+2.  On the **Main** menu, click **Identity \> Service Providers \>**
+    **Add**.
+3.  Fill in the **Service Provider Name** and provide a brief
+    **Description** of the service provider. Only **Service Provider
+    Name** is a required field and you can use Google-SP as the name for
```

Comment:
`Google-SP` -> this is not referred anywhere in the doc right? if the SP name is not used shall we use the register-a-service-provider.md fragment?